### ISSUE TITLE
Sorting fails when sorting on field of a related model 

### ISSUE THREAD

**Issue by nealtodd** [staticdev/django-sorting-bootstrap#9]
The implementation of [sort.py L7](https://github.com/staticdev/django-sorting-bootstrap/blob/master/sorting_bootstrap/sort.py#L7) prevents being able to sort by the field of a related model, resulting in a `KeyError: 'cl'` at L21.

E.g. given the models:

```
class Foo(models.Model):
    bar = models.ForeignKey(Bar)

class Bar(models.Model):
    baz = models.CharField()
```

sorting isn't possible on baz for a queryset of Foos, i.e.

```
{% auto_sort foos %}
...
{% sort_link 'Baz' 'bar__baz' %}
```

would raise the KeyError exception.

Rather than check whether the `sort_by` key is a direct field on the model and only order by it if that's the case, I can't see a reason not to use the EAFP principle and try to order by whatever `sort_by` is, and if that raises a field error then fall back to the logic in the `else` block. I.e. replace the `if-else` block by a `try-except` block.

Functionally, this would be doing exactly the same behaviour as currently happens, other than it would sort the queryset by the `sort_by` key if it's a valid argument to `order_by()` (such as a related field: `bar__baz` in my example).

The diff for [sort.py](https://github.com/staticdev/django-sorting-bootstrap/blob/master/sorting_bootstrap/sort.py) would be:

```
0a1,2
> from django.core.exceptions import FieldError
>
7c9
<         if sort_by in [el.name for el in queryset.model._meta.fields]:

---
>         try:
9c11
<         else:

---
>         except FieldError:
```

Happy to submit a PR for this if you'll accept it.

**Comment by nealtodd** [comment]
Hi @staticdev,

Is this still a maintained project?

It'd be good to get the diff in this issue incorporated if possible, with a new PyPi release.

(Py3 compatibility raised in #8, ahead of the Django 2.0 release which drops Py2 support, would be the icing on the cake for being able to continue using this package :)

If it's unmaintained [webstack-django-sorting](https://github.com/webstack/webstack-django-sorting/) may be the way to go as a replacement.

**Comment by camd** [comment]
@nealtodd Thanks for the suggestion on webstack-django-sorting.  This is a nice package, but I was hitting this error with an annotated field as well.  Same deal.

**Comment by staticdev** [comment]
@nealtodd @camd I hope it is now fixed on version 2.0.1. Sorry for long delay. I am looking for extra maintainers if you want. I have lots of code to maintain already.
